src/component/playground/PlaygroundResultsTable/ContextBanner/ContextBanner.tsx

```diff
@@ -0,0 +1,26 @@
+import { colors } from 'themes/colors';
+import { Alert, styled } from '@mui/material';
+import { SdkContextSchema } from 'openapi';
+
+interface IContextBannerProps {
+    context: SdkContextSchema;
+}
+
+const ContextFieldList = styled('ul')(() => ({
```

Comment:
Convention for styled components is `StyledContextFieldList` to indicate that this just adds styles to the component.